CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.15.0] - ???
```

Comment:
Is this minor enough to be 0.14.5 or will it be 0.15?

Reply:
It's a visible change in behavior so 0.15 I'd say